from app.exceptions import BusinessException


class UserAlreadyExists(BusinessException):
    pass
